### PR TITLE
Change Meta.load to return the simplified meta

### PR DESCRIPTION
**Pull request by rgrinberg**
It's always simplified anyway. If we're simplifying some things a late too eagerly, we could also make the simplification itself as lazy.

### CODE REVIEW

File: src/meta.ml [hunk 1]
```diff
@@ -1,16 +1,6 @@
 open Import
 
-type t =
```
Comment (diml) [inline src/meta.ml]: Why this change? I find it slightly nicer to have the main type first

Comment (rgrinberg) [reply src/meta.ml]: Just so that it's clear which  groups of types refer to each other. It's the 2nd commit, I can get rid of it.

Comment (diml) [reply src/meta.ml]: Ok, it's not a big deal I guess. But to be consistent `action` and `predicate` should be defined on their own.

Comment (rgrinberg) [reply src/meta.ml]: I just got rid of it, it's just a style change and I'm not sure it's applied consistently. I just thought you accidentally omitted it this one time.
